src/components/EditableFieldView.tsx

```diff
@@ -187,27 +188,29 @@ function EditableFieldView({
                     id="outlined-basic"
                     className={classes.textInput}
                     data-cy={`${cypressFieldNameFormatted}-text-field`}
-                    variant="outlined"
                     defaultValue={updatedFieldValue}
+                    disabled={isPendingSave}
```

Comment:
I was able to disable the text input for non-markdown `EditableFieldView`s while a save is pending, but for markdown `EditableFieldViews` (snapshot and dataset descriptions) we use a `SimpleMdeReact` component which does not allow for its textarea to be disabled.

I'm open to other ideas on how to handle that case but seeing as descriptions are modified as part of non-flight patch operations which return quickly, I elected to leave this alone… we will still get save and cancel buttons disabled while a save is pending.

Reply:
I agree, the save occurs pretty quickly so I don't think users would be able to edit the description through the UI while the save is pending